### PR TITLE
refactor

### PR DESCRIPTION
**Pull request by r-wheeler**
remove the explicit pattern matching 
remove unused `is_eager` argument
convert tensor to vector

### CODE REVIEW

File: src/tf.rs [hunk 1]
```diff
@@ -46,38 +36,38 @@ impl Servable for TensorflowModel {
...
+        // run the outout tensor
         let output_tensor = init_step.fetch::<f32>(output_token)?;
-        println!("{:?}", output_tensor[0]);
-        println!("{:?}", output_tensor[1]);
-        println!("{:?}", output_tensor[2]);
+
+        // convert to rust type
+        // TODO add trait magic to make output_tensor.as_vector() work
+        let mut dim = output_tensor.dims()[1] as usize;
+        let result: Vec<f32> = (0..dim).map(|i| output_tensor[i]).collect();
```
Comment (r-wheeler) [inline src/tf.rs]: 👀

**Comment by r-wheeler** [comment]
```
// Rust
trait Vectorizable {
    fn to_vector(&self) -> Vector<T>
}

impl Vectorizable for Tensor {
    fn to_vector(&self) -> Vector<T {
       dim = &sel.dims()
        ... 
    }
}
...
output_tensor.to_vector()
```

File: src/tf.rs [hunk 2]
```diff
@@ -46,38 +36,37 @@ impl Servable for TensorflowModel {
...
-                op_x = graph.operation_by_name_required("Placeholder")?;
-                op_y = graph.operation_by_name_required("head/predictions/probabilities")?;
-                op_init = graph.operation_by_name_required("init")?;
-            }
-            None => return utils::generate_err("Cannot set ops".to_string()),
-        };
 
+        // allocate the graph and session in local scope
+        let graph = self.graph.as_ref().ok_or("Cannot get graph")?;
+        let session = self.session.as_ref().ok_or("Cannot get session")?;
```
Comment (ericmcbride) [inline src/tf.rs]: dayum nice!

File: src/tf.rs (hunk 1 above)
Comment (ericmcbride) [reply src/tf.rs]: sic

**Pull request merged**